Fix the parse for a multipart email (#55)
demisto/parse-emails: CHANGELOG.md

# Changelog
v0.1.6

> * Fixed an issue where a multipart email is not being parsed properly.

Reviewer: ```suggestion
* Fixed an issue where a multipart email was not parsed properly.
```
